zephyr/lc3_config.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Bose Corporation All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LIB_LIBLC3CODEC_ZEPHYR_LC3_CONFIG_H_
+#define LIB_LIBLC3CODEC_ZEPHYR_LC3_CONFIG_H_
+
+/* This file is a c-wrapper for the c++ LC3 lib. Please refer to the C++ API documentation
+ * for a full description of the API.
+ * General usage:
+ * 1) Call _create to allocate memory for the c++ object using the c++-lib allocation mechanism.
+ * 2) Use the wrapper functions as needed passing the allocated object as first parameter.
+ * 3) After usage call _destroy to free up the memory.
+ *    Invalid usage of the object reference after a call to _destroy cannot be detected and will
+ *    most likely result in a crash. */
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+#define LC3_EXTERN_C extern "C"
+#else
+#define LC3_EXTERN_C
+#endif
+
+typedef void* Lc3Config_t;
+
+typedef enum
+{
+    lc3_duration_10ms,
+    lc3_duration_7p5ms
+} lc3_frame_duration_t;
+
+
+/**
+ * For detailed description please see underlying LC3 codec implementation.
+ * @param Fs_ Sample frequency HZ
+ * @param N_ms_ Frame duration
```

Comment:
Same answer - no clue - it is how they are used in the LC3 specification and the LC3 codec implementation.
That is why I felt the need to add a comment rather than only referring to the cpp documentation :-)

Reply:
See below